DS-Session4-Question3/src/meta/utility/CompoundMolicularMass.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This Class Only works For single Symbol Components till atomic no 19 elements
+ * @author Abhishek Bishnoi
+ * @since Aug 4, 2019
+ */
+package meta.utility;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Stack;
+
+public class CompoundMolicularMass {
+	LoadElements load = new LoadElements();
+	Map<Character,Integer> compunds = load.loadElements();
+	Stack<Integer> stack = new Stack<Integer>();
+	/**
+	 * Calculates mass of given compound Using Stack and Map Collection
+	 * @param compound is the organic compound
+	 * @return mass of the compound
+	 * Assumption Compunds's Symbols are Single Character Symbol till Atomic Number 19.
+	 */
+	public int calculateMass(String compound){
+		if(compunds==null){
+			System.out.println("Map Is Null");
+		}
+		try{
```

Comment:
You are using try block in order to catch EmptyStackException and then return 0 in the catch block. This will work fine when the parenthesis are properly matched. If there is some unevenly matched string is passed, eg: COH)2, then this method will return wrong output instead of throwing some exception, may be, "BracketsMisMatchException".